### PR TITLE
Release 2.13

### PR DESCRIPTION
**Pull request by Ahmadchurra681**
Please whitelisted my all code of conduct

### CODE REVIEW

**Comment by bk2204** [comment]
Hey,

It doesn't look like this is a helpful change, and you haven't explained why you want to do this in the PR message, so I'm going to close this.

**Comment by Alsabti5810** [comment]
> Please whitelisted my all code of conduct

**Comment by Ahmadchurra681** [comment]
Please white listed me

**Comment by Ahmadchurra681** [comment]
Please whitelisted me from all of my accounts i will pay all of my taxes which i can,t pay you

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -6,11 +6,15 @@ jobs:
     name: Build with default Git
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
+        os: [ubuntu-20.04, macos-latest]
```
Comment (SMG965FD) [inline .github/workflows/ci.yml:9]: .github/workflows/ci.yml

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -6,11 +6,15 @@ jobs:
     name: Build with default Git
     strategy:
       matrix:
-        os: [ubuntu-latest, macos-latest]
```
Comment (SMG965FD) [inline .github/workflows/ci.yml:9]: #4573

**Review by SMG965FD — COMMENTED** [review]
[.github/workflows/ci.yml]()

**Pull request closed without merging**